src/main/java/com/apachepoi/Reader.java

```diff
@@ -0,0 +1,93 @@
+package com.apachepoi;
+
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+
+public class Reader {
+    public static void read(String fileName) {
+        List<Employee> employees = new ArrayList<>();
+        try {
+            FileInputStream reader = new FileInputStream("src/main/resources/" + fileName);
+            Workbook workbook = null;
+            if (fileName.toLowerCase().endsWith("xlsx")) {
+                workbook = new XSSFWorkbook(reader);
+            } else if (fileName.toLowerCase().endsWith("xls")) {
+                workbook = new HSSFWorkbook(reader);
+            }
+            int numberOfSheets = workbook.getNumberOfSheets();
+            for (int i = 0; i < numberOfSheets; i++) {
+                Sheet sheet = workbook.getSheetAt(i);
+                Iterator iterator = sheet.rowIterator();
+                if (iterator.hasNext()) {
+                    iterator.next();
+                }
+                while (iterator.hasNext()) {
+                    Employee employee = new Employee();
+                    Row row = (Row) iterator.next();
+                    Iterator<Cell> cellIterator = row.cellIterator();
+                    while (cellIterator.hasNext()) {
+                        Cell cell = cellIterator.next();
+                        switch (cell.getColumnIndex()) {
+                            case 0:
```

Comment:
Don't we have a better logic instead of going with switch statement here?

Reply:
if else but its better with switch i thought...